pallets/parachain-staking/src/lib.rs

```diff
@@ -518,10 +519,15 @@ pub mod pallet {
 	pub(crate) type TotalSelected<T: Config> = StorageValue<_, u32, ValueQuery>;
 
 	#[pallet::storage]
-	#[pallet::getter(fn parachain_bond_info)]
+	#[pallet::getter(fn inflation_distribution_info)]
 	/// Parachain bond config info { account, percent_of_inflation }
-	pub(crate) type ParachainBondInfo<T: Config> =
-		StorageValue<_, ParachainBondConfig<T::AccountId>, ValueQuery>;
+	pub(crate) type InflationDistributionInfo<T: Config> = StorageMap<
```

Comment:
It's too complex and less efficient to use a StorageMap here.
Please go back to a StorageValue, and create instead a struct InflationDistributionConfig that contains all the field you need

Reply:
@TarekkMA also, the StorageMap imply more reads at round change block, more complex code in staking pallet that is already too complex, and more complex and costly benchmarks.
We need to reduce the complexity of this pallet, as it was the place of many critical bugs in the past, It is very important not to add unnecessary complexity!

Reply:
Will work on the new impl in this PR: #2976